google-http-client/src/main/java/com/google/api/client/http/GenericUrl.java

```diff
@@ -80,6 +80,9 @@
   /** Fragment component or {@code null} for none. */
   private String fragment;
 
+  /** Determines whether it should skip encoding/decoding parts of the url and use as is. */
```

Comment:
which is which? perhaps "If true use the URL as is; otherwise encode and decode it in ???? format"

Reply:
Done

Reply:
The Javadoc is not terminated; the code doesn't compile.